apps/web/package.json

```diff
@@ -24,12 +24,11 @@
     "node": ">=20.0.0"
   },
   "dependencies": {
+    "@latitude-data/database": "workspace:*",
     "@elastic/elasticsearch": "^8.14.0",
     "@t3-oss/env-nextjs": "^0.10.1",
     "dotenv": "^16.4.5",
-    "drizzle-orm": "^0.31.2",
     "next": "15.0.0-rc.0",
-    "pg": "^8.12.0",
```

Comment:
Drizzle ORM and PG runtime dependencies now lives in database. We call functions from this package. The convention is to put all the SQL calls under database/data-access